src/app/ffbe/mappers/effects/passives/passive-esper-summon-damage-increase.parser.ts

```diff
@@ -11,7 +12,21 @@ export class PassiveEsperSummonDamageIncreaseParser extends EffectParser {
     const esperId = effect[3][1];
     const esper = FFBE_ESPERS.find(e => e.gumiId === esperId);
```

Comment:
Cette ligne devrait être déplacée dans le second _else_ ci-dessous. Non seulement car c'est le seul endroit où la variable _esper_ est utilisée, mais surtout car tu as un conflit de nommage avec la variable _esper_ des l'itérations _esperId.map().filter()_ et _espers.map()_.

D'ailleurs je me demande pourquoi ton code compile. La variable _esper_ est mise en const à la ligne 13, elle ne devrait pas pouvoir être réutilisée aux lignes 21 et 22. C'est assez étrange quand j'y pense...